### PR TITLE
Finished html web page

### PR DESCRIPTION
**Pull request by PhillipLam89**
image

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+  <h1>Jelly of the Month Club</h1>
+
+  <h1>Make all your friends jelly of your stock of jelly!</h1>
+  <p>Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-quality and most unique jellies on a monthly basis. For the jelly lover in your</p>
+  <p>life (or yourself!), this is the gift that keeps on giving all year round. Wwe offer three tiers of monthly jelly toy, and each one offers their own perks. Here is a breakdown</p>
```
Comment (codingcodymiller) [inline html-web-page/index.html:27]: Though I understand the reasoning for this, when you have one large chunk of text like this, it is best to have it in one `p` element, and let the text wrap where it needs to based on the screen size 👍

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+      </tbody>
+      <tbody>
```
Comment (codingcodymiller) [inline html-web-page/index.html:48]: In a form, only one `tbody` is needed around the entire group of `tr` elements that would be in the table body.

File: html-web-page/index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        </tr>
+      </tbody>
+    </table>
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+
+    <label for="user-name">
+        <input id="user-name" type="text" placeholder="Your Name">
+    </label>
+
+    <label for="plans">
```
Comment (codingcodymiller) [inline html-web-page/index.html:87]: In this case, there is no label, so no `label` element is needed 👍

File: html-web-page/index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          <td>Three new curated jellies per month</td>
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+      </tbody>
+    </table>
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+
+    <label for="user-name">
```
Comment (codingcodymiller) [inline html-web-page/index.html:83]: In this case, there is no label, so no `label` element is needed 👍

File: html-web-page/index.html [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    <label for="plans">
+        <select name="dropdown" id="plans">
+          <option value="placeholder" >Select Your Plan</option>
+          <option value="option">Basic Plan</option>
+          <option value="option">Deluxe Plan</option>
+          <option value="option">Jelly Giant Plan</option>
+        </select>
+    </label>
+
+     <label for="submit"></label>
```
Comment (codingcodymiller) [inline html-web-page/index.html:96]: In this case, there is no label, so no `label` element is needed 👍

File: html-web-page/index.html [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+      </tbody>
+    </table>
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+
+    <label for="user-name">
+        <input id="user-name" type="text" placeholder="Your Name">
```
Comment (codingcodymiller) [inline html-web-page/index.html:84]: These form control elements are just hanging out free in the `body` element without being grouped together. If only there was a type of element we could use to wrap a group of form control elements to denote that those elements are meant to `form` one group of form control elements 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-web-page/index.html (hunk 6 above)
Comment (codingcodymiller) [reply html-web-page/index.html:77]: You missed a spot! Have you `form`ed an opinion on what type of element should wrap these form control elements? 😏

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-web-page/index.html (hunk 6 above)
Comment (PhillipLam89) [reply html-web-page/index.html:77]: omg.... fail  🤣

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**